<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearsDialog</class>
 <widget class="QDialog" name="GearsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvel équipement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="add_pushButton">
       <property name="text">
        <string> Ajouter équipement</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources/ressources.qrc">
         <normaloff>:/bike/edit.png</normaloff>:/bike/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbQuit">
       <property name="text">
        <string> Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources/ressources.qrc">
         <normaloff>:/bike/cancel.png</normaloff>:/bike/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbType">
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Marque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbMarque"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Modèle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leModele"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Date d'achat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deAchat">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Date de 1ère utilisation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deUtilisation">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>KM initial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leKMInitial">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Poids (g)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="lePoids">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labPrix">
           <property name="text">
            <string>Prix (€)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="lePrix">
           <property name="maximum">
            <double>9999999.990000000223517</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
